docs/tutorials/_index.yaml

```diff
@@ -61,9 +96,6 @@ landing_page:
         icon:
           path: /site-assets/images/cards/github-card-16x9.png
 
-    - heading: Intermediate
-      description: Use Cirq for intermediate-level subroutines and algorithms.
-      items:
```

Comment:
Would it be worthwhile to preserve these headings under the Github section? I can see how it might mess with formatting, though...

Reply:
It's definitely possible to do. I'll leave it to others for a final decision.

Reply:
+@balopat for review.